Makefile

```diff
@@ -10,6 +10,11 @@ docker-build-ldap:
 	docker pull debian:bullseye
 	cd ldap; \
 	docker build -t georchestra/ldap:${BTAG} .
+	
+docker-build-ldap-withrotation:
+	docker pull debian:buster
```

Comment:
Image pulled changed, please check.
Regarding new makefile target, it was required by @fvanderbiest , please check https://github.com/georchestra/georchestra/pull/3943/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52

I agree with him, since we need to add option if we need rotation password policy.

Reply:
> I agree with him, since we need to add option if we need rotation password policy.

But the option will be consumed at runtime, I'd have expected to have the same ldap docker image being generated, and the option at bootstrap which will decide if we want password rotation or not, this is why I don't get why we need a new makefile target (anyway since I don't use the makefile, I don't mind, if you both think this is needed, I'm fine)

Reply:
```
ARG PM_POLICY=default
ENV SLAPD_PASSWORD_MGT_POLICY ${PM_POLICY}
```
My understanding of these 2 lines from the PR (take it with a pinch of salt since I'm not actively involved in devs these days) is that we want to enable/disable password rotation at build time & at runtime, but we may miss some edge cases for the runtime change.

Indeed, considering the runtime change of the SLAPD_PASSWORD_MGT_POLICY option, we currently only deal with switching from "no password rotation" to "enabled password rotation". The symetrical use case does not seem to be dealt with.

Again, this is something I noticed from a very quick look at the code. Please bear with me if my findings are incorrect.

So, Pierre is right : if we have a working runtime option to switch back and forth between rotation / no rotation, there's no use having a specific makefile target.

Reply:
> But the option will be consumed at runtime

After a call with Marwane, we agreed to say that we do not intend to change the behavior at runtime.
It's build time **only**.